packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -174,15 +173,15 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
               elementToFocusOnDismiss={elementToFocusOnDismiss}
               isClickableOutsideFocusTrap={focusTrapZoneProps && !focusTrapZoneProps.isClickableOutsideFocusTrap ? false : true}
             >
-              <div ref={this._allowScrollOnPanel} className={_classNames.scrollableContent}>
+              <div className={_classNames.contentInner}>
```

Comment:
Is this 100% necessary. We're switching the location of the navigation to be inside of the contentInner. This could cause some regressions if others were expecting it to be outside.

Reply:
It is necessary to move the div. I don't know if it would be better to change the "contentInner" class name, as that would technically be a breaking interface change.

Reply:
Updated the styles, navigation is now back outside of contentInner.